ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/Impl/Util.hs

```diff
@@ -22,15 +23,20 @@ import           Data.Bifunctor (first)
 import           Data.List (sortOn)
 import           Data.Map.Strict (Map)
 import qualified Data.Map.Strict as Map
+import           Data.Proxy (Proxy (..))
 import           Data.Set (Set)
 import qualified Data.Set as Set
 import           Data.Text (Text)
 import qualified Data.Text as T
 import           Text.Read (readMaybe)
 
+import           Cardano.Prelude (Typeable)
```

Comment:
Import it from `Data.Typeable`, we'd like to drop the dependency on `cardano-prelude` in the future.